packages/orchestrator-ui-components/src/types/types.ts

```diff
@@ -319,6 +319,10 @@ export interface ProductDefinitionsResult {
     products: GraphQlResultPage<ProductDefinition>;
 }
 
+export interface ProcessStepsResult {
+    workflows: GraphQlSinglePage<{ steps: { name: Step['name'] }[] }>;
```

Comment:
See comment about type alias in `packages/orchestrator-ui-components/src/pages/processes/WfoStartProcessPage.tsx`

Try to extract an alias here